docs/app/pages/components/components-sections/buttons/floating-action-button/floating-action-button.component.html

```diff
@@ -0,0 +1,85 @@
+<ux-floating-action-buttons [ngClass]="direction" [direction]="direction">
+
+    <ux-floating-action-button fab-primary icon="hpe-contract" tooltip="Show Actions" container="body" [placement]="placement"></ux-floating-action-button>
+    <ux-floating-action-button icon="hpe-add" tooltip="Add New Item" container="body" [placement]="placement"></ux-floating-action-button>
+    <ux-floating-action-button icon="hpe-analytics" tooltip="Show Analytics" container="body" [placement]="placement"></ux-floating-action-button>
+    <ux-floating-action-button icon="hpe-app" tooltip="Show Overview" container="body" [placement]="placement"></ux-floating-action-button>
+
+</ux-floating-action-buttons>
+
+<hr>
+
+<div class="row uxd-customize-example">
+    <div class="col-md-12">
+        <accordion>
+            <accordion-panel heading="Customize Example...">
+                <div class="row">
+                    <div class="col-md-3">
+                        <ux-radio-button [(ngModel)]="direction" option="top">Top</ux-radio-button>
```

Comment:
Can you change this to refer to the API values, so that authors can directly associate it with the code that needs to be written?

````
direction    ( ) top   (x) right   ( ) bottom   ( ) left
````